stubs/resources/views/resource-collections/create.blade.php

```diff
@@ -0,0 +1,27 @@
+<x-app-layout>
+    <x-slot name="header">
+        <h1>
+            {{ __('resourceCollection.create_title') }}
+        </h1>
+    </x-slot>
+
+    <!-- Form Validation Errors -->
+    @include('partials.validation-errors')
+    <form action="{{ localized_route('resource-collections.store') }}" method="POST" novalidate>
+        @csrf
+        <x-hearth-input id="user_id" type="hidden" name="user_id" :value="Auth::user()->id" required />
+
+        <div class="field @error('title') field--error @enderror">
+            <x-hearth-label for="title" :value="__('resourceCollections.label_title')" />
```

Comment:
Throughout the views added in this pull request, the translation strings are referenced with a plural key in camelCase (`resourceCollections`). It should be a singular, kebab-case key (`resource-collection`).